### PR TITLE
1264 docs per SID

### PR DESCRIPTION
**Pull request by mwfrost**
This builds on the approved branch for ticket #1264 , and uses the same ajax endpoint.

The 1264 branch has already been merged into this one.

As the ticket specifies, I have removed the table of documents per keyword set from the page, but the `keysets` object is still in the `response.context` for a chemical.

### CODE REVIEW

File: dashboard/tests/functional/test_ajax.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import json
+
+from django.test import TestCase, override_settings
+from dashboard.tests.loader import fixtures_standard
+
+
+@override_settings(ALLOWED_HOSTS=["testserver"])
+class TestProductList(TestCase):
```
Comment (michael-barbour) [inline dashboard/tests/functional/test_ajax.py]: Looks like this is a test case for products but later on you hit endpoints /d_json/ and /c_json/ (documents and chemicals respectively).

File: dashboard/tests/functional/test_ajax.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    def test_anonymous_read(self):
+        response = self.client.get("/products/")
+        self.assertEqual(response.status_code, 200)
+
+    def test_default_params(self):
+        response = self.client.get("/p_json/")
+        self.assertEqual(response.status_code, 200)
+        data = json.loads(response.content)
+        self.assertEquals(data["recordsTotal"], 35)
```
Comment (michael-barbour) [inline dashboard/tests/functional/test_ajax.py]: Asserting a hard-coded number seems odd to me.  It might be better to replace this with an aggregate of all expected products `Product.objects.all().aggregate(Count('pk'))`.  If this value is going to be used repeatedly, saving it to `self` as a variable would allow us to test against it with minor changes.

This would allow us to add more products to seed data without expecting all these hard-coded values to have to change.

File: templates/puc/puc_detail.html [hunk 3]
```diff
@@ -114,8 +114,7 @@ <h4>Products Assigned To "{{ puc }}"</h4>
             <tbody></tbody>
         </table>
 
-    </div>
```
Comment (michael-barbour) [inline templates/puc/puc_detail.html]: This div looks like it was removed on accident.
